src/main/java/com/powsybl/openloadflow/network/LfBranch.java

```diff
@@ -45,6 +47,10 @@
 
     double getPermanentLimit2();
 
+    Map<Integer, Double> getTemporaryLimits1();
```

Comment:
It's a bit strange to use a map, I think returning a List<TemporaryLimit> would be clearer.

Reply:
I commited with a `SortedSet<CurrentLimits.TemporaryLimit>` instead, as we're only using it sorted anyway. Using this class means that the values are not in PerUnit though.